src/poetry/installation/chef.py

```diff
@@ -42,10 +42,6 @@ def is_wheel(self, archive: Path) -> bool:
         return archive.suffix == ".whl"
 
     def get_cached_archive_for_link(self, link: Link) -> Optional[Link]:
-        # If the archive is already a wheel, there is no need to cache it.
-        if link.is_wheel:
-            pass
-
```

Comment:
I wonder what's the intention of this `pass` was? Maybe this should be a `return`?

Reply:
According to the comment, it looks like that. I would prefer to make a new issue about it and not mix two things at the same time?

Reply:
It's probably a good idea to handle this in a different issue because it would have impact on how things work. I would prefer keeping the current lines of code in this PR.

Reply:
This makes sense, I will change it back